npm package "lighthouse" is not installed globally or locally

Started to get this error on a Release pipeline for Azure DevOps 

```
2023-02-10T10:01:46.5101000Z ##[section]Starting: Lighthouse https://*** Mobile
2023-02-10T10:01:46.6598492Z ==============================================================================
2023-02-10T10:01:46.6599254Z Task         : Lighthouse
2023-02-10T10:01:46.6599713Z Description  : Display Google Lighthouse reports and evaluate audit scores with assertions.
2023-02-10T10:01:46.6600172Z Version      : 1.0.6
2023-02-10T10:01:46.6600817Z Author       : GSoft
2023-02-10T10:01:46.6601361Z Help         : You can provide the lighthouse NPM package, otherwise it will be installed in a temporary directory.
2023-02-10T10:01:46.6601691Z ==============================================================================
2023-02-10T10:01:47.7665695Z Node.js found at: C:\Program Files\nodejs\node.exe
2023-02-10T10:01:47.7915425Z NPM found at: C:\Program Files\nodejs\npm.cmd
2023-02-10T10:01:47.7928982Z Temporary directory: D:\a\_temp\__lighthouse
2023-02-10T10:01:47.7941220Z Lighthouse target URL: https://***
2023-02-10T10:01:47.7950742Z Working directory: D:\a\r1\a
2023-02-10T10:01:47.8051931Z Lighthouse HTML report will be saved at: D:\a\_temp\__lighthouse\***-15639.report.html
2023-02-10T10:01:47.8053160Z Lighthouse JSON report will be saved at: D:\a\_temp\__lighthouse\***-15639.report.json
2023-02-10T10:01:47.8540225Z Existing Lighthouse installation not found
2023-02-10T10:01:47.8541313Z Lighthouse will be installed using NPM at: D:\a\_temp\__lighthouse
2023-02-10T10:01:47.8586709Z [command]C:\Windows\system32\cmd.exe /D /S /C ""C:\Program Files\nodejs\npm.cmd" install lighthouse --prefix D:\a\_temp\__lighthouse "--loglevel=error""
2023-02-10T10:02:26.4108165Z 
2023-02-10T10:02:26.4118508Z added 128 packages, and audited 129 packages in 35s
2023-02-10T10:02:26.4122962Z 
2023-02-10T10:02:26.4125598Z 17 packages are looking for funding
2023-02-10T10:02:26.4132358Z   run `npm fund` for details
2023-02-10T10:02:26.4134277Z 
2023-02-10T10:02:26.4139549Z found 0 vulnerabilities
2023-02-10T10:02:26.4595560Z 
2023-02-10T10:02:26.4684743Z Installing Lighthouse with NPM returned: 0
2023-02-10T10:02:26.4730471Z ##[error]npm package "lighthouse" is not installed globally or locally
2023-02-10T10:02:26.4759225Z Lighthouse task finished
```

--- a/task/src/library.ts
+++ b/task/src/library.ts
@@ -328,9 +328,10 @@ export class LighthouseTask {
   }
 
   private getLocallyInstalledLighthouseExecPath(): string {
-    const nodeModulesPath = path.join(this.workingDirectory, 'node_modules');
-    const execPath = path.join(nodeModulesPath, 'lighthouse', 'lighthouse-cli', 'index.js');
-    return fs.existsSync(execPath) ? execPath : '';
+    const execPathPostLH10 = path.join(this.workingDirectory, 'node_modules', 'lighthouse', 'cli', 'index.js');
+    const execPathPreLH10 = path.join(this.workingDirectory, 'node_modules', 'lighthouse', 'lighthouse-cli', 'index.js');
+
+    return fs.existsSync(execPathPostLH10) ? execPathPostLH10 : fs.existsSync(execPathPreLH10) ? execPathPreLH10 : '';
   }
 
   private getGloballyInstalledLighthouseExecPath(): string {
@@ -339,8 +340,11 @@ export class LighthouseTask {
   }
 
   private async locallyInstallAndGetLighthouseExecPath() {
-    const execPath = path.join(this.tempDirectory, 'node_modules', 'lighthouse', 'lighthouse-cli', 'index.js');
-    if (fs.existsSync(execPath)) return execPath;
+    const execPathPostLH10 = path.join(this.tempDirectory, 'node_modules', 'lighthouse', 'cli', 'index.js');
+    const execPathPreLH10 = path.join(this.tempDirectory, 'node_modules', 'lighthouse', 'lighthouse-cli', 'index.js');
+
+    if (fs.existsSync(execPathPostLH10)) return execPathPostLH10;
+    if (fs.existsSync(execPathPreLH10)) return execPathPreLH10;
 
     console.log('Existing Lighthouse installation not found');
     console.log(`Lighthouse will be installed using NPM at: ${this.tempDirectory}`);
@@ -350,7 +354,7 @@ export class LighthouseTask {
     const npmResultCode = await npmCommand.exec();
     console.log(`Installing Lighthouse with NPM returned: ${npmResultCode}`);
 
-    return fs.existsSync(execPath) ? execPath : '';
+    return fs.existsSync(execPathPostLH10) ? execPathPostLH10 : fs.existsSync(execPathPreLH10) ? execPathPreLH10 : '';
   }
 
   private async executeLighthouse() {

